pages.ko/common/git-am.md

```diff
@@ -0,0 +1,16 @@
+# git am
+
+> 패치 파일을 적용한다. 이메일로 커밋을 받을 때 유용함. 패치 파일을 생성 할 수 있는 `git format-patch` 또한 참고.
```

Comment:
```suggestion
> 패치 파일을 적용한다. 이메일로 커밋을 받을 때 유용함.
> 패치 파일을 생성 할 수 있는 `git format-patch` 또한 참고.
```
I guess this would be better and would make this page better aligned with main page. What do you think?

Reply:
Suggestion addressed and updated 👍 Thank you @kbdharun